runtime/onert/odc/MinMaxReader.cc

```diff
@@ -99,6 +100,10 @@ MinMaxVectors MinMaxReader::readOP(uint32_t model_idx, uint32_t subg_idx, uint32
   float minmax[2];
   const int64_t data_size = sizeof(float) * 2 + sizeof(uint32_t) * 3;
 
+  // Check num_run overflow
+  if (num_run > std::numeric_limits<int32_t>::max() / data_size)
```

Comment:
Can I get the reason why the max value of `int32_t` is used?

Reply:
I've updated to `uint32_t`.

Reply:
I was curious about how to check for overflow with `std::numeric_limits<int32_t>::max() / data_size`.
Anyway, adding code to check for overflow looks good.

Reply:
Analyzer required to check upper limit for loop condition from file reading. So I added upper limit based on file size limit.